### PR TITLE
Homepage improvements

### PR DESCRIPTION
**Pull request by zachlatta**
This pull request makes a variety of improvements to the homepage based on feedback I've been receiving from people that have seen it.

Notably:

- A bunch of copy changes
- Reorganizing the photos in "Meet the Hack Clubs"
- Moving the workshops, resources, and supporters sections to their own "How It Works" page (which is currently incomplete and not linked to from anywhere).
- Simplifying navbar

@LabLayers: can I have you take a glance at this? If you give me a LGTM, I'll go ahead and merge to make these changes live.

### CODE REVIEW

**Comment by zachlatta** [comment]
I'm actually just going to go ahead and merge this because we don't have any other full-time contributors to the website.

@LabLayers: I'm still very keen to hear your feedback, if you have any. Just comment on the PR and I'll make sure to keep up to date.

**Comment by LabLayers** [comment]
It makes sense to move a lot of the content to a separate page since the homepage was a bit lengthy - good call. :+1:

**Comment by LabLayers** [comment]
There may be a bit of confusion (or lack of information) on what Hack Club provides for clubs, which happens to be one of our major reasons for starting a club.

I suggest adding a link to the new page called "Our Resources" in the navbar and an extra FAQ item linking to that page

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -83,28 +82,37 @@
             class: 'card-image-fx-normal'
           = image_tag 'slices/resources-red.svg',
             class: 'card-image-fx-hover'
-        %h3.card-heading.card-heading-larger Hack Club's Resources
+        %h3.card-heading.card-heading-taller Our Resources
```
Comment (LabLayers) [inline app/views/pages/home.html.haml]: Get rid of .card-heading-taller from all 4 card headers now that "Our Resources" can fit on one line.

**Comment by LabLayers** [comment]
Other than that, I like the minor design and content changes.

**Pull request merged**